packages/react/src/patterns/sub-patterns/TableOfContents/TOCDesktop.js

```diff
@@ -64,12 +64,25 @@ const TOCDesktop = ({ menuItems, selectedId, updateState }) => {
     });
     const title = filteredItems[0].title;
     updateState(id, title);
+    triggerFocus(`a[name="${id}"]`);
     document.querySelector(`a[name="${id}"]`).scrollIntoView({
       behavior: 'smooth',
       block: 'start',
     });
   };
 
+  /**
+   * Trigger the focus on screen readers, so they can read the target paragraph
+   *
+   * @param {*} elem Selector to find the item
+   */
+  function triggerFocus(elem) {
+    const element = document.querySelector(elem);
+    element.setAttribute('tabindex', '0');
+    element.focus({ preventScroll: true });
+    element.removeAttribute('tabindex');
+  }
+
```

Comment:
We should apply this to the utility function. The more add here to fix bugs the further the two become away from each other.

Reply:
So we should turn this into a utility?